.github/workflows/build.yaml

```diff
@@ -9,9 +9,42 @@ on:
     - master
 
 jobs:
-  hivedscheduler:
-    name: hivedscheduler
+  test:
+    name: Test on Go-${{ matrix.go }}-${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        go: [1.12.x, 1.13.x, 1.14.x]
+        os: [ubuntu-16.04, ubuntu-latest]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: src/github.com/${{ github.repository }}
+      - name: Setup go ${{ matrix.go }}
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+      - name: Setup GOPATH
+        run: echo "::set-env name=GOPATH::${{ github.workspace }}"
+      - name: Go build
+        run: |
+          cd src/github.com/${{ github.repository }}
+          go build -o hivedscheduler ./cmd/hivedscheduler/
+      - name: Go test
```

Comment:
Call go build by go-build.sh

Reply:
seems full go build is already included in packaging/building docker image pipe, the build in this pipeline is just to generate a coverage report

Reply:
But the coverage is generated in the go test, isn't it?